CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # Change Log
 
+## [v1.2.0](https://github.com/ably/ably-ruby/tree/v1.2.0)
+
```

Comment:
Is it worth including the 'Full Changelog' link here, per the previous release?

Reply:
Good spot 👍 [64fd0cf](https://github.com/ably/ably-ruby/pull/349/commits/64fd0cff6840f5091bb63af780bfa73bd2867d7f)